### ISSUE TITLE
RAID tests ?

### ISSUE THREAD

**Issue by vincegre** [koct9i/ioping#54]
Hi

Is it usable to test read/write speed on a RAID system ? or only for individual drives ?

Thanks

Vincèn

**Comment by koct9i** [comment]
Yes. RAID devices and fs on top of them provides exactly the same interface as other block devices and fs. This tool issues standard file I/O operations and measures time.
